client-app/core/composables/useErrorsTranslator.test.ts

```diff
@@ -0,0 +1,267 @@
+import { describe, expect, it, vi } from "vitest";
+import { useErrorsTranslator } from "./useErrorsTranslator";
+
+function mockI18n(): void {
+  vi.mock("vue-i18n", async (importOriginal) => {
+    const actualI18n = await importOriginal();
+    return {
+      ...{ actualI18n },
+      useI18n: vi.fn().mockReturnValue({
+        t: (key: string) => key,
+        te: (key: string) => key,
+      }),
+    };
+  });
+}
+
+describe("use-errors-translator", () => {
+  mockI18n();
+
+  const { localizedItemsErrors, setErrors } = useErrorsTranslator("validation_error");
```

Comment:
test cases should to be isolated, the new instance of `useErrorsTranslator` should be created for each test case